websocket/websocket-server.go

```diff
@@ -0,0 +1,335 @@
+package websocket
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+	"github.com/gorilla/websocket"
+
+	runtime "github.com/Hammad887/chat-app"
+	domain "github.com/Hammad887/chat-app/models"
+)
+
+// ctx is a variable of type context.Context, which is used to manage request-scoped values, deadlines, and cancellations.
+var ctx context.Context
```

Comment:
why we are using this context here, it should be passed from the main file , life we are using the swagger server

Reply:
Using context.Background() in the main file.